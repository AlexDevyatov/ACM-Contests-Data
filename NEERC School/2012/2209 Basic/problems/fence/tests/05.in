7
-28 25
-22 8
-10 -25
29 -23
30 2
29 11
24 27
17 9
