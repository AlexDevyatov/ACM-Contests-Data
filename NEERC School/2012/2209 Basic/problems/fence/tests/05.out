8
-28 25
-22 8
-10 -25
29 -23
30 2
17 9
29 11
24 27
